src/containers/track-overflow-modal/ConnectedMobileOverflowModal.tsx

```diff
@@ -140,7 +141,7 @@ const ConnectedMobileOverflowModal = ({
           onUnfavorite: () => unsaveTrack(id as ID),
           onShare: () => shareTrack(id as ID),
           onAddToPlaylist: () => addToPlaylist(id as ID, title),
-          onVisitTrackPage: () => visitTrackPage(id as ID, handle, title),
+          onVisitTrackPage: () => visitTrackPage(permalink || ''),
```

Comment:
is the || '' dangerous at all? maybe the fn should just be 

() => permalink ? visitTrackPage(...) : null

Reply:
in theory it shouldn't be null I'll change the types

Reply:
JK I can't, because these props are also used for other things besides tracks. I'll put a console error there?

Reply:
👍  sounds like a typing issue we should eventually clean up though